### PR TITLE
Use Bootstrap theme

### PR DESCRIPTION
**Pull request by mroest**
In order to create a consistent look and feel for all the
GSSP plugins the UI is updated to use the bootstrap framework.

Other changes:
* Removed language selector. Language selection is handled by Stepup Gateway
* Upgrade symfony / flex recipes to support composer 2.0
* Upgrade CSS / JS dependencies to resolve encore compat issues and recent security vulns

### CODE REVIEW

**Comment by phavekes** [comment]
Production build fails on :

```
[vagrant@stepup-build /vagrant]$ /usr/bin/php72 /usr/local/bin/composer install --prefer-dist --ignore-platform-reqs --no-dev --no-interaction --optimize-autoloader
Installing dependencies from lock file
Verifying lock file contents can be installed on current platform.

                                                             
  [ErrorException]                                           
  constant(): Couldn't find constant SODIUM_LIBRARY_VERSION 
```

Travis build has issues too:

```
> symfony-cmd
sh: 1: symfony-cmd: not found
Script symfony-cmd handling the auto-scripts event returned with error code 127
Script @auto-scripts was called via post-install-cmd
The command "composer install" failed and exited with 127 during .
```

Could you take a look at this?

**Review by MKodde — COMMENTED** [review]
I'm not terribly stoked on all the changes from the updated recipes, but that is the price you pay for using a framework in all its glory I guess.

See one nitpick below you might want to address. @phavekes can you test-drive this change?

File: package.json [hunk 1]
```diff
@@ -1,21 +1,28 @@
...
+        "core-js": "^3.0.0",
+        "css-loader": "^5.0.0",
+        "import-glob-loader": "^1",
+        "jquery": "^3",
+        "node-sass": "^4.14.1",
+        "regenerator-runtime": "^0.13.2",
+        "sass-loader": "^8",
+        "webpack-notifier": "^1.6.0"
+    },
+    "license": "UNLICENSED",
```
[the remark this answers is not in the document]
Comment (mroest) [reply package.json]: Ah yes, updated

**Comment by mroest** [comment]
> I'm not terribly stoked on all the changes from the updated recipes, but that is the price you pay for using a framework in all its glory I guess.

Yes, i could leave it out but next time you update recipes it will be much worse and time consuming to figure out what needs to stay or go...

**Review by phavekes — APPROVED** [review]

File: public/index.php [hunk 2]
```diff
@@ -12,11 +12,11 @@
     Debug::enable();
 }
 
-if ($trustedProxies = $_SERVER['TRUSTED_PROXIES'] ?? $_ENV['TRUSTED_PROXIES'] ?? false) {
+if ($trustedProxies = $_SERVER['TRUSTED_PROXIES'] ?? false) {
```
[the remark this answers is not in the document]
Comment (mroest) [reply public/index.php:15]: That was removed due to the symfony/flex recipe update. Not sure if this was a local/custom change. If so i can put it back to be sure.... Let me know.

[the remark this answers is not in the document]
Comment (phavekes) [reply public/index.php:15]: I don't see any current uses of TRUSTED_PROXIES in .env files, so it's safe to leave out here.

**Pull request merged**